### PR TITLE
Rebalances dormant diseases and sample crates

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
dormant diseases should be about twice as common, and have better symptoms
virus sample crates have had their cost increased from 3750 to 6000

## Why It's Good For The Game
Dormant diseases are far less likely to appear than intended currently
virus sample crates are still king for virologists

this pr should mitigate that somewhat

## Testing Photographs and Procedure
image
it compiles, but this is a probability change, largely

## Changelog
:cl:
balance: dormant diseases should now be more common, virus sample crate price has significantly increased
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Why exactly is limiting viro's access to the crates good for the game? As a non viro player I don't really know the use case

**Comment by zeskorion** [comment]
> Why exactly is limiting viro's access to the crates good for the game? As a non viro player I don't really know the use case

virus sample crates are extremely powerful compared to other disease sources. There are around 30 individual symptom rolls per crate, each of which can be level 9. considering virus bounties are consistently worth 10000+ credits, a virologist can get around 2-3 symptom crates easily with a sizeable  profit to cargo, and get the stuff they want. Even at 6000 crates are still desirable, but you cant easily get enough crates to find any symptom you could want

**Comment by EvilDragonfiend** [comment]
The core reason why you buy viro crates is to get 9 lvl virus, and if you can spam buying the crate because your station is rich and budgetful, you don't have to maint hunt to get 9 lvl virus.

File: code/modules/jobs/job_types/brig_physician.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 	species_outfits = list(
 		SPECIES_PLASMAMAN = /datum/outfit/plasmaman/brig_physician
 	)
-	biohazard = 15 //still deals with the sick and injured, just less than a medical doctor
+	biohazard = 25 //still deals with the sick and injured, just less than a medical doctor
```
Comment (itsmeow) [inline code/modules/jobs/job_types/brig_physician.dm:30]: these changes are undocumented

Comment (zeskorion) [reply code/modules/jobs/job_types/brig_physician.dm:30]: this is the variable that determines dormant disease chance

Comment (EvilDragonfiend) [reply code/modules/jobs/job_types/brig_physician.dm:30]: It seems Kat was confused with a single CL line that might have to be two lines for two changes.

**Review by itsmeow — APPROVED** [review]

**Pull request merged**